test/box/graceful_shutdown.lua

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env tarantool
+local os = require('os')
+
+box.cfg{
+    listen              = os.getenv("LISTEN"),
+    memtx_memory        = 107374182,
```

Comment:
Why are you need to specify memtx_memory?

Reply:
Removed unnecessary options.